src/CSModule.sol

```diff
@@ -62,10 +62,12 @@ contract CSModule is
     ////////////////////////
     // State variables below
     ////////////////////////
-    /// @dev DEPRECATED. Moved to CSParametersRegistry
-    uint256 internal _keyRemovalCharge;
+    mapping(uint256 queuePriority => QueueLib.Queue queue) queueByPriority;
 
-    QueueLib.Queue public depositQueue;
+    /// @dev Legacy queue (priority=10), that we will probably may be able to remove in the future.
+    QueueLib.Queue public legacyQueue;
+
+    uint256 private constant LEGACY_QUEUE_PRIOIRITY = 10;
```

Comment:
Why not make it zero and put the others on top? In the current approach, you effectively limit the number of priority queues. It might work only if we want to have de-prioritized queues, but do we?

Reply:
Like for big NOs with even less priority to enter?

Reply:
Yeah. But I think we want to keep the default one with the lowest priority. Hence, the proposal is to make it's priority = 0 and sort priorities in the descending orger

Reply:
Changed the design.